grid/chat/threads/profile.supr

```diff
@@ -0,0 +1,170 @@
+"""
+Chat Threads
+Retrieve chat threads in the workspace.
+"""
+name = "chat/threads"
+version = "1.0.0"
+
+"""
+Get Threads
+Retrieve paginated list of threads
+"""
+usecase GetThreads {
+  input  {
+    """
+    Workspace
+    Representation of a workspace. Can be guild or workspace, depending on the provider capability.
+    """
+    workspace! string!
+  }
+
+  result {
+    """
+    Threads
+    List of retrieved threads
+    """
+    threads! [Thread]
+
+    """
+    Rate Limit
+    Rate limit details.
+    """
+    rateLimit RateLimit
+  }
+
+  error Error
+
+  example Successful {
+    input {
+      workspace = 'workspace_id'
+    }
+
+    result {
+      threads = [
+        {
+          id = '1'
+          createdAt = 1546167104000
+          channel = 'random'
+        },
+        {
+          id = '2'
+          createdAt = 1546167105000
+          channel = 'random'
+        }
+      ]
+    }
+  }
+
+  example Failed {
+    input {
+      workspace = 'not-existing-workspace'
+    }
+
+    error {
+      title = 'workspace_not_found'
+    }
+  }
+}
+
+model Thread {
```

Comment:
If possible, consider adding fields with messages count and the time of the latest message (e.g. `updatedAt` or `lastMessageAt`).